examples/slurm/README.md

```diff
@@ -0,0 +1,47 @@
+Moneo Job Id Update Integration w/ Slurm
+=====
+Description
+-----
+Moneo allows for the assignment of job IDs to a subset of nodes. The command ```python3 moneo.py -j <jobid> -c <job host.ini file>``` accomplishes this. The command requires a host.ini file which has a specific format. It also requires a job Id.
+
+The host.ini has the following format:
+```
+[worker]

```

Comment:
Can we avoid the need for "[worker]" tag in the file?

This will allow users to specify the hostfile directly without having to edit it.

Reply:
I'll investigate what can be done. I will track this in the other comment. Resolving this comment.